Fakebook.Posts/Fakebook.Posts.Domain/Models/Comment.cs

```diff
@@ -1,8 +1,18 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Text;
 
-namespace Fakebook.Posts.Domain {
-    public class Comment {
+namespace Fakebook.Posts.Domain.Models
+{
+    public class Comment
```

Comment:
a comment probably needs a post to be valid but i'll just let this through as is